pages/introduction/about/README.md

```diff
@@ -91,21 +91,22 @@ Besides the diligent online community, a group of people regularly physically ga
 
 <!-- ![The core team working on the Accessibility Developer Guide (details read below)](_media/core-team.jpg) -->
 
-- **Andreas Moesch**, web developer, Nothing AG.
-- **Donato Rotunno**, web developer, Liip AG.
-- **Esther Brunner**, web developer, Zeix AG.
-- **Graciela Schütz**, senior UX architect, UNIC AG.
-- **Joshua Muheim**, web developer & accessibility expert, «Access for all».
+- **Andreas Moesch**, web developer, Nothing.
+- **Christoph Dubach**, UX architect & web developer, UNIC.
+- **Donato Rotunno**, frontend developer & accessibility expert, Liip.
+- **Esther Brunner**, interface developer, Zeix.
+- **Graciela Schütz**, UX architect, UNIC.
+- **Joshua Muheim**, frontend developer & accessibility expert, Accessify&#46;me.
```

Comment:
@christophdubach, out of curiosity: What is the reason for using `&#46;` over `.`?

Reply:
Using a `.` turns the substring into a link. And if responsive.ch and Accessify.me become links, then it would be fair to link _all_ company names.

Therefore, I have decided not to use links.